docs/BasicCommunication/OnSystemTimeReady/index.md

```diff
@@ -0,0 +1,37 @@
+## OnSystemTimeReady
+
+Type
+: Notification
+
+Sender
+: HMI
+
+Purpose
+: Notifies SDL that the HMI is ready to provide accurate system time
+
+!!! must
+
+The HMI must send a BC.OnSystemTimeReady notification to SDL when a new ignition cycle occurs in order to notify SDL that the HMI is once again ready to provide accurate system time.
+
+!!!
+
+### Notification
+
+#### Parameters
+
+`This RPC has no additional parameter requirements.`
```

Comment:
No need for backticks

Reply:
removed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/94/commits/a3d94936ec418984d4c144090b48b649d3279632